app/src/main/java/org/fossasia/pslab/fragment/ExperimentDocMdFragmentNew.java

```diff
@@ -0,0 +1,73 @@
+package org.fossasia.pslab.fragment;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.widget.NestedScrollView;
+import android.view.LayoutInflater;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.widget.ImageView;
+
+import org.fossasia.pslab.PSLabApplication;
+import org.fossasia.pslab.R;
+
+import br.tiagohm.markdownview.MarkdownView;
+
+/**
+ * Created by ThisIsNSH on 4/15/2018.
+ */
+
+public class ExperimentDocMdFragmentNew extends Fragment {
+
+    private NestedScrollView scrollView;
+    private String mdFile, mdFile1, imagePath;
```

Comment:
@ThisIsNSH I meant in these cases. We can name them `mdFileTop`, `mdFileBottom` and so on!

Reply:
👍